test/e2e_node/critical_pod_test.go

```diff
@@ -56,6 +58,8 @@ var _ = framework.KubeDescribe("CriticalPod [Serial] [Disruptive] [NodeFeature:C
 			if !configEnabled {
 				framework.Skipf("unable to run test without dynamic kubelet config enabled.")
 			}
+			// because adminssion Priority enable，If the priority class is not found, the Pod is rejected.
```

Comment:
the `，` here is a https://www.fileformat.info/info/unicode/char/ff0c/index.htm
please use `,` instead.